### ISSUE TITLE
Review manual 

### ISSUE THREAD

**Issue by jmshapir** [gslab-econ/ra-manual#9]
[gslab](https://github.com/gslab-econ/ra-manual/wiki/Clusters)
[gentzkow](https://github.com/gentzkow/ra-manual/wiki/Workflow)

**Comment by jmshapir** [comment]
*  [Getting Started](https://github.com/gslab-econ/ra-manual/wiki/Getting-Started) -- looks like main change is dropping pointers to where to learn about various types of software. I'm fine with that.
  *  Computing Environment
      *  [Workstations](https://github.com/gslab-econ/ra-manual/wiki/Workstations) -- I'd think this is useful to have but it's more on the Stanford side so defer.
      *  [Storage](https://github.com/gslab-econ/ra-manual/wiki/Storage) -- probably good to keep rules re: file sizes in github, those are important. No strong feelings on the dropbox stuff. Since everything else in this header is pretty Stanford-specific, could segment this into a different place.
      *  [Clusters](https://github.com/gslab-econ/ra-manual/wiki/Clusters) -- seems like moved to an appendix, no objections to that.
      *  [IT Support](https://github.com/gslab-econ/ra-manual/wiki/IT-Support) -- Stanford specific so defer on where to maintain. 
  *  Project Management
      *  [Projects](https://github.com/gslab-econ/ra-manual/wiki/Projects) -- probably obvious enough that we can drop.
      *  [Tasks](https://github.com/gslab-econ/ra-manual/wiki/Tasks) -- There's some stuff re: comment threads, prioritizing work in the gslab version that we'd want to keep.
      *  [Sprints](https://github.com/gslab-econ/ra-manual/wiki/Sprints) -- Legacy, can drop
      *  [Mothball](https://github.com/gslab-econ/ra-manual/wiki/Mothball) -- Seems like moved to an appendix, fine with that
  *  Version Control
      *  [Repository Organization](https://github.com/gslab-econ/ra-manual/wiki/Repository-Organization) -- ideally redundant with template
      *  [Commit](https://github.com/gslab-econ/ra-manual/wiki/Commit) -- seems like can drop since this is just github worfklow?
      *  [SCons](https://github.com/gslab-econ/ra-manual/wiki/SCons) -- maybe drop because redundant with scons documentation?
  *  Other Collaboration Tools
      *  [Google Calendar](https://github.com/gslab-econ/ra-manual/wiki/Google-Calendar) -- could probably update this since we mainly use it for OOF
      *  [Evernote](https://github.com/gslab-econ/ra-manual/wiki/Evernote) -- do you still want to use this?
      *  [Paperpile](https://github.com/gslab-econ/ra-manual/wiki/Paperpile) -- do we need a page about this?
  *  Coding Principles
      *  [Code Style](https://github.com/gslab-econ/ra-manual/wiki/Code-Style) -- might be good to keep some of the "good citizen" bits & keeping it short. Fine to drop the rest.
      *  [Unit Testing](https://github.com/gslab-econ/ra-manual/wiki/Unit-Testing) -- could probably drop
      *  [Data Storage](https://github.com/gslab-econ/ra-manual/wiki/Data-Storage) -- probably good to keep in some form
      *  [Data Structure](https://github.com/gslab-econ/ra-manual/wiki/Data-Structure) -- maybe not needed since covered in code & data?
  *  [Paper Production](https://github.com/gslab-econ/ra-manual/wiki/Paper-Production) -- pretty well maintained and I think identical between two versions
  *  [PhD Application](https://github.com/gslab-econ/ra-manual/wiki/PhD-Application) -- pretty well maintained and I think identical between two versions
  *  Appendix A: Style Guides
      *  [Paper Drafts](https://github.com/gslab-econ/ra-manual/wiki/Paper-Drafts) -- probably good to keep this in some form so we have a default for everything. 
      *  [Data Visualization](https://github.com/gslab-econ/ra-manual/wiki/Data-Visualization) -- since short maybe we can fold it in elsewhere, or maybe we can just make a single page that points to the key resources we use as style references, with special pages for things like Stata and C where our style manual is more extensive
      *  [R](https://github.com/gslab-econ/ra-manual/wiki/R)
      *  [Python](https://github.com/gslab-econ/ra-manual/wiki/Python)
      *  [Stata](https://github.com/gslab-econ/ra-manual/wiki/Stata)
      *  [C](https://github.com/gslab-econ/ra-manual/wiki/C)
	  *  [Template and GSLab Python Development](https://github.com/gslab-econ/ra-manual/wiki/Template-and-GSLab-Python-Development) -- would probably fold into template / library readme
  *  [Appendix B: Legacy Tools](https://github.com/gslab-econ/ra-manual/wiki/Appendix-B:-Legacy-Tools) -- probably good to keep in some form since this does sometimes come up

**Comment by jmshapir** [comment]
@gentzkow I took a look over the gslab manual and the gentzkow flavor. Above are my notes, organized according to the sidebar in the gslab version. Let me know what you think is a good next step.

**Comment by gentzkow** [comment]
Thanks! 

I agree w/ most of these notes. A lot of them are things I tried to implement in the new version. Can you give me a sense of what changes if any we'd need to make to the new version to make it acceptable to you?

**Comment by jmshapir** [comment]
@gentzkow I tried to do that in the notes above. Basically, I started with the gslab version and commented on my thoughts on what happened in the other version. In some cases there is no counterpart in which case I commented on whether I think we should keep/drop/modify.

**Comment by jmshapir** [comment]
(Happy to do a zoom w screen share sometime if you think that would be more efficient.)

**Comment by gentzkow** [comment]
OK, sure. Here's my attempt to extract a to do list from your summary. Let me know if this looks right or if there are things to add / subtract. Goal = once to do list is complete we can flip to the new manual.

1. Move Stanford-specific content on workstations / storage / IT support not currently in MG version to appendix to the extent it seems worth keeping
2. From "Tasks", keep stuff re: comment threads, prioritizing work in either the main "Issues" page or in the GSLab-specific appendix section
3. Keep Google Calendar and Evernote (if we decide to use it) in GSLab-specific appendix section
4. Add some of the "good citizen" bits from code style to the "Code" page
5. Keep "Data Storage" in some form (MG: unsure here because most of this is currently in the "Data" section; let me know what you'd like to add)
6. Move "paper drafts" material not covered on "Papers & Slides" tab to a new "Papers"  appendix page under "Style Guides"
7. Move "data visualization" material to a new "Figures" appendix page under "Style Guides"
8. Fill in "Legacy Tools" page w/ existing material

**Comment by jmshapir** [comment]
Thanks. A few notes:

* On (2) actually I think the comments stuff from tasks looks pretty similar between the two so probably it's just the bit about prioritizing that we need to port over?
* The page called "storage" is different from the one called "data storage." From the "storage" one, we want to make sure we keep the rules on what we store / how we store stuff in github. Those seem important.
* On (4), would also keep the "keeping it short" part of "code style"
* On (5), I think we should swap out readme.txt for readme.md. And we should probably note that the goal of the /docs part is to ensure that even if we lose physical access to the original data source (say because a link breaks) we have enough documentation to be able to interpret the data, reference it in a paper, etc. The goal of making each raw directory as self-contained as possible seems like an important thing to emphasize.
* On (7), I'm not sure if it's worth a whole page for this. An alternative would be to have a single "style guide" page that in some cases points to external resources (e.g. Schwabish) and in some cases points to other pages (e.g. our Stata style guide)
* We might want to keep the "C" style guide too.
* Before we cut over, we should probably ask @yu-jia-wang @Reese565 to review to see if we've missed anything that we want to port.

**Comment by gentzkow** [comment]
Perfect! Will implement.

(On (7), I might be inclined to make a page for it on the view that it would be a good idea to keep fleshing out best practices for figures. But I don't feel strongly either way.)

**Comment by gentzkow** [comment]
Procedurally, I will open a task in the gentzkow flavor RA manual repo to implement, then open a task for the predoc team to do another round of review and flip the switch.

**Comment by gentzkow** [comment]
See [here](https://github.com/gentzkow/ra-manual/issues/1)

**Comment by jmshapir** [comment]
Summary: In this issue we reviewed differences between the gslab and gentzkow flavors of the RA manual wiki and made a plan for revisions to harmonize them. Notes on differences [here](https://github.com/gslab-econ/ra-manual/issues/9#issuecomment-776145258); plan summarized [here](https://github.com/gslab-econ/ra-manual/issues/9#issuecomment-776241776) and [here](https://github.com/gslab-econ/ra-manual/issues/9#issuecomment-776284485). Next steps [here](https://github.com/gslab-econ/ra-manual/issues/9#issuecomment-776338008).
